composer.json

```diff
@@ -7,9 +7,9 @@
     "require-dev": {
```

Comment:
How specific do you think we should be with version numbers? I am used to locking things as precisely as possible, but we have a very heavy emphasis on reproducible builds at my workus, so I really haven't ever tried it another way.